docs/likelihood.rst

```diff
@@ -204,14 +204,24 @@ central value equal to unity for each bin in the channel. The scales
 uncertainties of samples defined within the channel relative to the total event
 rate of all samples: :math:`\delta_{csb} = \sigma_{csb}/\sum_s \nu^0_{scb}`. As
 not all samples are within a channel are estimated from MC simulations, only
-the samples with a declared statistical uncertainty modifier enter the sum. An
-example is shown below:
+the samples with a declared statistical uncertainty modifier enter the sum.
+An example of a statistical uncertainty modifier for a single bin channel is
+shown below:
 
 .. code:: json
 
    { "name": "mod_name", "type": "staterror", "data": [0.1] }
 
-An example of a statistical uncertainty modifier.
+.. warning::
+
+   For bins in the model where:
+
+     * the samples nominal expected rate is zero, or
+     * the scale factor is zero.
+
+   nuisance parameters will be allocated, but will be fixed to ``1`` in the
+   calculation (as staterror is a multiplicative modifier this results in
+   multiplying by ``1``).
```

Comment:
This is the added warning for staterror. Basically the warning from shapesys but with "scale factor" instead of "absolute uncertainty" to describe the modifier data.